spec/controllers/items_controller_spec.rb

```diff
@@ -34,46 +33,73 @@
   end
 
   context "new item is created successfully" do
+    before(:each) do
+      user =  User.create(username: "dreamteam", email: "us@dream.com", password_hash: "zzzzzzz")
```

Comment:
1) Pull this into a let
2) Move the line below to a method called login_user that does this. The name will explain what it is doing.
3) Call `login_user(user)` in your before block
4) Just use your `user` in your let anywhere you are doing `User.last`. Querying the last user is a less clear way of getting the specific user that you want to test with.